firebase-vertexai/src/main/kotlin/com/google/firebase/vertexai/Chat.kt

```diff
@@ -49,11 +51,17 @@ public class Chat(
   private var lock = Semaphore(1)
 
   /**
-   * Generates a response from the backend with the provided [Content], and any previous ones
-   * sent/returned from this chat.
+   * Sends a message using the existing history of this chat as context and the provided [Content]
+   * prompt.
    *
-   * @param prompt A [Content] to send to the model.
-   * @throws InvalidStateException if the prompt is not coming from the 'user' role
+   * If successful, the message and response will be added to the history. If unsuccessful, history
+   * will remain unchanged.
+   *
+   * This function should be called within a suspend context to properly manage concurrency.
+   *
```

Comment:
Kind of a NOP, since it's a `suspend` function; it can only be ran from a suspend context.

```suggestion
```